.travis.yml

```diff
@@ -1,6 +1,15 @@
+sudo: required
```

Comment:
Nit: this is already at the end of the file, but is it still required now that you've removed line 26?